python/unityagents/socket_communicator.py

```diff
@@ -0,0 +1,95 @@
+import logging
+import socket
+import struct
+
+
+from communicator import UnityRLOutput, UnityRLInput,\
+    UnityOutput, UnityInput,AcademyParameters,\
+    UnityInitializationInput, UnityInitializationOutput
+from .exception import UnityTimeOutException
+
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger("unityagents")
+
+
+class SocketCommunicator(object):
```

Comment:
Shouldn't these two classes be inheriting a common interface, especially since they have some common code?

Reply:
Will do !